packages/simulator/tests/simulator.spec.ts

```diff
@@ -1,3 +1,30 @@
+import { WalletAPIClient } from "@ledgerhq/wallet-api-client";
+import { getSimulatorTransport, profiles } from "../src";
+
 describe("Simulator", () => {
-  it.todo("Should write simulator tests");
+  describe("bitcoin.getXPub", () => {
+    it("should return the xpub", async () => {
+      const transport = getSimulatorTransport(profiles.STANDARD);
+      const client = new WalletAPIClient(transport);
+
+      const xPub = await client.bitcoin.getXPub("accountId");
+
+      expect(xPub).toBe("xpub");
+    });
+
+    it("should throw an error if permission not granted", async () => {
+      const profile = {
+        ...profiles.STANDARD,
+        permissions: {
+          ...profiles.STANDARD.permissions,
+          methodIds: [],
+        },
+      };
```

Comment:
Permission is done at the top level so it should maybe be done one time only at the upper level
We might want to test the following thing:
- Should throw a NOT_IMPLEMENTED ServerError if not implemented in wallet handlers
- Should throw a new type of ServerError if the choosen account is not from the appropriate family (granularity of the error to be discussed with the team)